elementary/monitor/dbt_project/macros/get_new_source_freshness_alerts.sql

```diff
@@ -0,0 +1,45 @@
+{% macro get_new_source_freshness_alerts(days_back) %}
+    -- depends_on: {{ ref('alerts_source_freshness') }}
+    {% set elementary_database, elementary_schema = elementary.target_database(), target.schema %}
+
+    {% set select_new_alerts_query %}
+        with new_alerts as (
+            select * from {{ ref('alerts_source_freshness') }}
+            where alert_sent = false and {{ elementary.cast_as_timestamp('detected_at') }} >= {{ get_alerts_time_limit(days_back) }}
+        ),
+        sources as (
+            select * from {{ ref('elementary', 'dbt_sources') }}
+        )
+
+        select *
+        from new_alerts
+        join sources on new_alerts.unique_id = sources .unique_id
```

Comment:
why do you need this join if alerts_source_freshness already joins the sources to the source freshness results?

Reply:
Thanks! That was before I did it in the alerts model.